gateway/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "gateway-simulator"
+version = "0.1.0"
+authors = ["sysarcher <sysarcher@users.noreply.github.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+pancurses = "0.16.1"
+rand = "0.8.4"
+#kafka = "0.8"
```

Comment:
if not used, please remove it or provide reason why it should stay

Reply:
done with the next commit